client/auth/login.jsx

```diff
@@ -73,6 +108,11 @@ export class Auth extends Component {
 	handleChange = ( event ) => {
 		const { name, value } = event.currentTarget;
 
+		if ( name === 'auth_code' ) {
+			this.setState( { auth_code: value.replace( /\s/g, '' ) } );
```

Comment:
This has one consequence that is not horrible, but probably unintended: the `this.state.auth_code` value is also used to set the value of the form input, and the spaces will never appear in the input field, although you are typing them.

An alternative would be to set the state as provided by the `change` event, and sanitize the value only right before using it as parameter for `makeRequest`.